cogs/moderation.py

```diff
@@ -174,6 +175,8 @@ async def set_mute(self, ctx, muterole: discord.Role):
     @command(name='mute', aliases=['silence', 'deafen', 'tempmute'])
     async def mute(self, ctx, member: discord.Member, duration: Optional[GlobalChannel], *, reason: str):
         with Repository() as repo:
+            guild = repo.get_guild(ctx.guild.id)
+            
             muterole = repo.get(Role, and_(
```

Comment:
This bit is outdated from when roles still had toggles for their features, now instead you should use:
```suggestion
            muterole = guild.config.mute_role
```